src/qt/guiutil.cpp

```diff
@@ -820,7 +823,7 @@ QString loadStyleSheet()
     QSettings settings;
     QString cssName;
     QString theme = settings.value("theme", "").toString();
-
+    
```

Comment:
Small Nit: empty line with whitespace characters

Not a huge deal right now, as there are many such instances throughout the code and I do plan to do a sweeping cleanup in the future, but lets try and minimize these going forward.